detox/src/devices/drivers/AndroidDriver.js

```diff
@@ -199,11 +200,16 @@ class AndroidDriver extends DeviceDriverBase {
   async _launchInstrumentationProcess(deviceId, bundleId, rawLaunchArgs) {
     const launchArgs = this._prepareLaunchArgs(rawLaunchArgs);
     const additionalLaunchArgs = this._prepareLaunchArgs({debug: false});
+    const serverPort = new URL(this.client.configuration.server).port;
+    this.adb.reverse(deviceId, serverPort)
     const testRunner = await this.adb.getInstrumentationRunner(deviceId, bundleId);
     const spawnFlags = [`-s`, `${deviceId}`, `shell`, `am`, `instrument`, `-w`, `-r`, ...launchArgs, ...additionalLaunchArgs, testRunner];
 
     this.instrumentationProcess = spawnAndLog(this.adb.adbBin, spawnFlags, { detached: false });
-    this.instrumentationProcess.childProcess.on('close', () => this._terminateInstrumentation());
+    this.instrumentationProcess.childProcess.on('close', () => {
+      this._terminateInstrumentation()
```

Comment:
Could you rewrite to that to `async`-`await` style, please?
![async-await-style](https://i.imgur.com/7MS015k.png)